app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -82,6 +83,30 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     }
   };
 
+  listenToRx(function(data) {
+    if (data.controller !== 'catalogItemFormController') {
+      return;
+    }
+
+    if (data.key) {
+      $timeout(function() {
+        // get ids of checked additional tenants in the tree
+        var new_ids = miqTreeObject('tenants_tree').getChecked().map(getTenantId);
+        new_ids = new_ids.filter(tenant_id => tenant_id); // filter unnecessary root node if present
```

Comment:
can't use `=>` in `app/assets/javascripts`.

Unless you want to move this under `app/javascript`, this has to be

```js
         // filter unnecessary root node if present
         new_ids = new_ids.filter(function(tenant_id) {
          return tenant_id;
         });
```